website/scripts/build-static-content.js

```diff
@@ -293,12 +293,17 @@ module.exports = {
                 await sails.helpers.fs.write(htmlOutputPath, htmlString);
               }
 
+              // Determine the path of the docs folder in the fleet repo so we can link to
+              // it from the documentation on fleetdm.com (e.g. 1-Using-Fleet/2-fleetctl-CLI.md)
+              let pathInDocsFolder = path.relative(path.join(topLvlRepoPath, sectionRepoPath), path.resolve(pageSourcePath));
+
               // Append to what will become configuration for the Sails app.
               builtStaticContent.markdownPages.push({
                 url: rootRelativeUrlPath,
                 title: pageTitle,
                 lastModifiedAt: lastModifiedAt,
                 htmlId: htmlId,
+                pathInDocsFolder: pathInDocsFolder,
```

Comment:
can we name this key something like `sectionRelativeRepoPath`?  (to future-proof for handbook pages and others)